client/src/ui/components/trading/MarketOrderPanel.tsx

```
@@ -500,17 +500,17 @@ const OrderCreation = ({
-    setBid((lords / resource).toFixed(2));
+    setBid(String(lords / resource));
   }, [resource, lords]);
 
   const updateLords = useCallback((newBid: number, newResource: number) => {
-    setLords(Number((newBid * newResource).toFixed(2)));
+    setLords(Number(newBid * newResource));
   }, []);
 
   const handleBidChange = (newBid: number) => {
     const numericBid = Number(newBid);
     if (!isNaN(numericBid) && numericBid > 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)